suite-native/module-passphrase/src/components/PassphraseScreenHeader.tsx

```diff
@@ -23,6 +23,11 @@ import {
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Translation } from '@suite-native/intl';
 import { cancelPassphraseAndSelectStandardDeviceThunk } from '@suite-native/passphrase';
+import TrezorConnect from '@trezor/connect';
+
+type PassphraseScreenHeaderProps = {
+    shouldRemoveCurrentPassphraseWallet: boolean;
```

Comment:
I would add onClose to the prop name for better understandability. Something like:
```suggestion
    isWalletRemovedOnClose: boolean;
```

Reply:
https://github.com/trezor/trezor-suite/pull/12677/commits/5dde9a246eb412fc29d8f12f75138271543c371b